sycl/source/detail/device_impl.hpp

```diff
@@ -30,6 +30,8 @@ using PlatformImplPtr = std::shared_ptr<platform_impl>;
 // TODO: Make code thread-safe
 class device_impl {
 public:
+  static constexpr pi_handle_type handle_type = PI_NATIVE_HANDLE_DEVICE;
```

Comment:
```suggestion
  static constexpr pi_handle_type PIHandleType = PI_NATIVE_HANDLE_DEVICE;
```
Or just HandleType, please apply to all similar places.

Reply:
Good call. I've renamed them to `PIHandleType`.